src/assets/sass/protocol/components/_feature-card.scss

```diff
@@ -0,0 +1,214 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+/* -------------------------------------------------------------------------- */
+// A single large feature card with media, headline & description.
+
+.mzp-c-card-feature {
+    margin: 0 auto $margin-xl;
+    text-align: center;
+
+    // Preserve aspect ratio of images and avoid content jump when loading.
+    &.mzp-has-aspect-16-9 .mzp-c-card-feature-media {
+        @include aspect-ratio(16, 9);
+    }
+
+    &.mzp-has-aspect-3-2 .mzp-c-card-feature-media {
+        @include aspect-ratio(3, 2);
+    }
+
+    .mzp-c-card-feature-media {
+        overflow: hidden;
+        position: relative;
+    }
+
+    .mzp-c-card-feature-media {
+        img,
+        video {
+            height: auto;
+            width: 100%;
+        }
+    }
+
+    .mzp-c-card-feature-content {
+        @include bidi(((text-align, left, right),));
+        margin-top: $margin-md;
+    }
+
+    .mzp-c-card-feature-title {
+        @include text-display-sm;
+    }
+
+    .mzp-c-card-feature-desc {
+        @include text-body-md;
+        color: $color-gray-70;
+        margin: $margin-xs 0;
+    }
+
+    .mzp-c-card-feature-list li {
+        margin-bottom: $margin-xl;
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+    }
+
+    // dark theme color styling
+    .mzp-t-dark & {
+        color: $color-white;
+
+        .mzp-c-card-feature-desc {
+            color: $color-gray-30;
+        }
+    }
+
+    @media #{$mq-md} {
+        margin-bottom: $layout-2xl;
+        max-width: 800px;
+
+        .mzp-c-card-feature-list li {
+            margin-bottom: $margin-2xl;
+        }
+
+        &.mzp-l-card-feature-left-half,
```

Comment:
Right now this is just based on the existing naming convention that we used for the billboard component. I'm all for finding something better, but changing the class name would likely be a breaking change and something we should do uniformally for all components, so I think this is probably best saved for a separate issue. 

I do like `mzp-l-reverse`, but that wouldn't quite work here as the default layout is centered. We'd need to come up with something that can be applied to both orientations independently.

Reply:
Right, I wanted to at least bring it up before merging this, since if we change our approach we would also need to update the billboard (and maybe some other places). OR if we decide to commit to using left and right in our already-presentational layout classes I might want to update the hero section to follow that convention.

So let's stay with left and right for the moment (at least where it's already been used) but we should decide soon if we want to change that or commit to it forever.